app/src/main/java/com/evanfuhr/pokemondatabase/activities/MainActivity.java

```diff
@@ -95,6 +96,14 @@ public void onClick(View view) {
                 onClickButton(view);
             }
         });
+
+        _natureButton = (Button) this.findViewById(R.id.buttonNatureList);
+        _natureButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                onClickButton(view);
```

Comment:
So am I reading this correctly? The view specific to `_natureButton` passes its view to the `onClickButton()` and process its case to deliver the intent for NatureListActivity.class?

Reply:
Yes. The names are not particularly descriptive.